examples/kind/kind-expose-port.yaml

```diff
@@ -4,6 +4,10 @@ nodes:
 - role: control-plane
 - role: worker
   extraPortMappings:
-  - containerPort: 8080
-    hostPort: 8080
+  - containerPort: 80
```

Comment:
Were the original 8080 ports incorrect? As far as I understand it Contour configures the Envoy containers to listen on 8080 and 8443.

Reply:
No, the 8080 ports worked ok; I'd changed to 80 to align with blog post examples and other documentations.
Would be happy to change back to 8080 and add 8443 and redo docs for consistency.